autoload/vimlparser.vim

```diff
@@ -3950,6 +3955,17 @@ function! s:ExprParser.parse_expr8() abort
         endif
       endif
       unlet node
+    elseif token.type ==# s:TOKEN_ARROW
+      let node = s:Node(s:NODE_METHOD)
+      let node.pos = token.pos
+      let node.left = left
+      let node.right = self.parse_expr8()
+      let node.lambda_rlist = s:NIL
+      if node.right.type !=# s:NODE_CALL
+        throw s:Err('Invalid method syntax', node.right.pos)
```

Comment:
Can this error have more info?  Or is it enough?

Reply:
fixed 14221a4e2b21c66f7dd4550be8934cea3f7d3f6c